pkgs/top-level/all-packages.nix

```diff
@@ -15910,6 +15910,8 @@ with pkgs;
 
   thinkingRock = callPackage ../applications/misc/thinking-rock { };
 
+  nylas-mail = callPackage ../applications/networking/mailreaders/nylas-mail { };
```

Comment:
Please ping me when you do — GitHub PR update notifications are either buggy or non-existent, not sure which